library/src/main/kotlin/com/connectrpc/ProtocolClientConfig.kt

```diff
@@ -81,7 +83,10 @@ class ProtocolClientConfig @JvmOverloads constructor(
     // is returned, the entire call must complete before it elapses. If the
     // call is still active at the end of the timeout period, it is cancelled
     // and will result in an exception with a Code.DEADLINE_EXCEEDED code.
-    val timeoutOracle: TimeoutOracle = { null },
+    //
+    // The default oracle, if not configured, returns a 10 second timeout for
+    // all operations.
+    val timeoutOracle: TimeoutOracle = { 10.toDuration(DurationUnit.SECONDS) },
```

Comment:
I added this to account for _removal_ of the timeouts in the configuration helper below. So if users use `ConnectOkHttpClient.configureClient` and forget to set timeouts in this config, they get the same default behavior as the `OkHttpClient` was providing (except that this default applies to bidirectional streams, whereas the `OkHttpClient` timeouts do not).